### PR TITLE
[WIP] Street Election 2020 Website

### PR DESCRIPTION
**Pull request by sigmachirality**
Nothing of note has been done yet tbh

### CODE REVIEW

**Review by sigmachirality — COMMENTED** [review]
why was "index.js" deleted in the election components folder? I know some of the other project pages don't use an index.js page, but I think it's a superior way of organizing a folder especially if you're going to be doing multiple imports from the same folder, just generally cleaner

File: .prettierrc [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
   "arrowParens": "avoid",
-  "semi": false
+  "semi": false,
+  "singleQuote": true
```
[the remark this answers is not in the document]
Comment (peterbaile) [reply .prettierrc:4]: fk u, can u rerun ur `gatsby format` shit?

**Comment by peterbaile** [comment]
@laurellee133 plz follow the Trello card and rewrite DTao's code

File: .prettierrc (hunk 1 above)
[the remark this answers is not in the document]
Comment (sigmachirality) [reply .prettierrc:4]: just run npx gatsby format LOL

**Comment by peterbaile** [comment]
nice job @laurellee133 💯 I think for the next step we hv to think abt how to generate the JSON files automatically using a script

**Review by peterbaile — APPROVED** [review]

**Pull request merged**